### ISSUE TITLE
[Bug] - Crowd Fluentd Configuration Does Not Match Other Charts

### ISSUE THREAD

**Issue by orangewolf** [atlassian/data-center-helm-charts#504]
### Suggestion

configmap-fluentd.yaml in most of the charts contain:
```
  {{ if .Values.fluentd.customConfigFile }}
{{- range $key, $value := .Values.fluentd.fluentdCustomConfig }}
  {{ $key }}: |
{{ $value | indent 4 }}
  {{- end }}
```

this value is missing from the Crowd version, even though Crowd values file and README seem to indicate that customConfigFile and fluentdCustomConfig are supported.

Ideally, this charts configmap-fluentd.yaml would be made to match the other charts.

### Product

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nanux** [comment]
Hi @orangewolf, currently, the Crowd helm chart doesn't have the same level of official support as the other products, as it was a community contribution. We are considering changing this status in our roadmap, and if that is the case, we will align the Crowd helm chart with the rest of the products, including the fluentd config.

Thank you

**Comment by orangewolf** [comment]
@nanux would you consider a PR for this if I put one up? I'd much rather do that than maintain a fork and my current work around (deploy once with Helm and then update the configmap and restart) is not really tenable.

**Comment by nanux** [comment]
@orangewolf, we are welcoming PRs. Feel free to raise the PR and we will review it.

**Comment by bianchi2** [comment]
Fixed and is going to be in 1.12.0 release
